web/src/components/SortButton/SortButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+import {
+  Select,
+  SelectContent,
+  SelectGroup,
+  SelectItem,
+  SelectLabel,
+  SelectSeparator,
+  SelectTrigger,
+  SelectValue,
+} from '../ui/select'
+
+const SortButton = () => {
+  return (
+    <Select>
+      <SelectTrigger className="w-[210px]">
+        <SelectValue placeholder="Sorteren" />
+      </SelectTrigger>
+      <SelectContent>
+        <SelectGroup>
+          <SelectLabel>Sorteren op</SelectLabel>
+          <SelectSeparator />
+          <SelectItem value="latest">Laatste</SelectItem>
```

Comment:
I was wondering if rephrasing it to make it super clear. Something like, newest first vs oldest first. What do you think? 

```suggestion
          <SelectItem value="newestFirst">Nieuwste Eerst</SelectItem>
```

Reply:
"Meest recent" may also be a good idea